lib/suite.js

```diff
@@ -183,7 +183,8 @@ export default class Suite extends EventEmitter {
                 port: suiteConfigData.port,
                 scheme: suiteConfigData.scheme,
                 base_path: suiteConfigData.base_path,
-                read_timeout: suiteConfigData.read_timeout
+                read_timeout: suiteConfigData.read_timeout,
+                common_headers: suiteConfigData.common_headers,
```

Comment:
This is not required, because we are already reading common_headers from YAML on line#268